spec/index.html

```diff
@@ -2723,14 +2725,14 @@ <h2>Serialization</h2>
   <p>When serializing quads in <a>canonical n-quads form</a>,
     components which are <a>blank nodes</a> MUST be serialized using the
     canonical label associated with each <a>blank node</a>
-    from the <a>map</a> component of the <a>normalized dataset</a>.</p>
+    from the <a>map</a> component of the <a>canonicalized dataset</a>.</p>
```

Comment:
We might want to be more clear here since there are two map components:

```suggestion
    from the <a>issued identifiers map</a> component of the
    <a>canonicalized dataset</a>.</p>
```